xblock/mixins.py

```diff
@@ -538,3 +538,50 @@ def index_dictionary(self):
         default implementation is an empty dict
         """
         return {}
+
+
+class ViewsMixin(object):
+    """
+    This mixin provides decorators that can be used on xBlock view methods.
+    """
+    @classmethod
+    def supports(cls, functionality):
```

Comment:
Nice idea.  I'd like to be consistent with the other xBlock decorators.  It looks like `@needs` and `@wants` take only a single service name.  Should we update those as well if we like the list of strings approach?